IPs/RSKIP171.md

```diff
@@ -0,0 +1,113 @@
+# Title
+
+|RSKIP          |171           |
+| :------------ |:-------------|
+|**Title**      |Clean EVM Internal Buffer in Call-like Opcodes |
+|**Created**    |02-09-2020 |
+|**Author**     |FJ |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+The purpose of this RSKIP is to be fully compatible with [EIP-211](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-211.md). `RETURNDATACOPY` and `RETURNDATASIZE` opcodes have already been implemented in previous RSKIPs, but still, there were some cases for call-like opcodes needed to be implemented.
```

Comment:
is to be -> is to make RSK

Reply:
"some cases for call-like opcodes needed to be implemented" -> "some corner cases related to call-like opcodes that weren't implemented in previous network upgrades"

Reply:
https://github.com/rsksmart/RSKIPs/pull/171/commits/b0a1a5b19faa78111ddc3a46972366c474cedb87